src/EnliteMonolog/Service/MonologServiceAwareInterface.php

```diff
@@ -9,15 +9,7 @@
 
 interface MonologServiceAwareInterface
 {
+    public function setMonologService(Logger $monologService): void;
 
-    /**
-     * @param Logger $monologService
-     * @return void
-     */
-    public function setMonologService(Logger $monologService);
-
-    /**
-     * @return Logger
-     */
-    public function getMonologService();
+    public function getMonologService(): ?Logger;
```

Comment:
why nullable? when would an implementation of this interface not return a Logger from this method?

Reply:
Oh, my [comment](https://github.com/enlitepro/enlite-monolog/pull/68#issuecomment-871906081) references this question.